usr/lib/python2.7/site-packages/xapi/storage/libs/libcow/datapath.py

```diff
@@ -303,14 +273,12 @@ class QdiskDatapath(COWDatapath):
     @staticmethod
     def parse_uri(uri):
         # uri will be like:
-        # "vhd+tapdisk://<sr-type>/<sr-mount-or-volume-group>|<volume-name>"
+        # "qdisk://<sr-type>/<sr-mount-or-volume-group>|<volume-name>"
```

Comment:
You always replaced `vhd+tapdisk` by `tapdisk` in this PR, here `qdisk`, why?

Reply:
It's an error in the imported code.
The `libcow` contains code relative to Qcow2, QEMU... The uri scheme name must be `qdisk`, not `vhd+tapdisk`.